src/test/e2e/Helpers/Selectors.ts

```diff
@@ -9,8 +9,9 @@ module.exports = {
   SearchInput: '#search-input',
 
   // Home selectors
-  StartNowButton: '.govuk-button--start',
-  MainHeader: 'h1.govuk-heading-xl',
+  StartNowButton: '.govuk-button--start', //unused currently, to be re-used if start page reverts.
```

Comment:
id bin this off, if its not used, odds are were going to be sticking with this for now

Reply:
sorted.